frontend/lib/norent/data/faqs-content.tsx

```diff
@@ -574,3 +576,26 @@ export const FaqsContent: Faq[] = [
     answerFull: <CollectiveOrganizing />,
   },
 ];
+
+export type FaqWithPreviewOptions = Faq & {
+  previewOptions: FaqPreviewOptions;
+};
+
+/**
+ * A list of all FAQs with preview options, pre-sorted to reflect
+ * their priority.
+ */
+export const FaqsWithPreviewContent: FaqWithPreviewOptions[] = [];
+
+for (let faq of FaqsContent) {
+  const { previewOptions } = faq;
+  if (previewOptions) {
+    FaqsWithPreviewContent.push({ ...faq, previewOptions });
+  }
```

Comment:
Super weird that this appears to be the most TypeScript-friendly way to create this array.  You'd think we could just say `if (faq.previewOptions) FaqsWithPreviewContent.push(faq)` but that doesn't work. 😞

Reply:
Huh, odd!